LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -4282,7 +4282,7 @@ Description="Some ADVENT robots and some other units gain the Damage Control abi
 [DarkEvent_HardTarget X2DarkEventTemplate]
 DisplayName="Tactical Upgrades: Hard Target"
 SummaryText="Some alien units gain the Hard Target ability, granting bonus dodge."
-QuoteText="\".\""
+QuoteText="\"The aliens' reflexes are becoming adept in our environment. We should invest in our scopes and make sure no one leaves without one.\""
```

Comment:
Would you mind putting these lines around it so that translators know that they need to update the other language files?
```
; LWOTC Needs Translation
QuoteText=...
; End Translation

Reply:
#589 0ec80b55c813d852c99661447238f8eb83fecaae

Done! (I think) How is it?

Reply:
Erm...can you make it exactly "; LWOTC Needs Translation"? It means folks can just search for "LWOTC Needs Translation" to find the bits that they need to work on.

Reply:
Also, I like it to be searchable case sensitively, so capitalisation is important.

Reply:
c2f71de51ad12beaee5d913909e5d7fbd2c31eb7

Right, totally understandable since coding is very specific and precise. It is my first time joining in a github project to contribute, but hopefully after a few more times, I got this!